nucypher/utilities/seednodes.py

```diff
@@ -1,14 +1,14 @@
 
 
-from json.decoder import JSONDecodeError
-
-import json
-
-import os
-
-from typing import Set, Optional, Dict, List
-
-from nucypher.config.constants import DEFAULT_CONFIG_ROOT
+# from json.decoder import JSONDecodeError
```

Comment:
At this point, this entire file is commented out and therefore unused. If you can delete it, and the test suite passes, then let's just delete it.

Reply:
Yep! I figured there must be a good reason for it still existing and didn't dare get rid!

Reply:
Heh, no good reason as far as I know. You have the green light to remove🚦- just blame it on me if any issue subsequently comes up 😅